lucene/core/src/java/org/apache/lucene/index/StoredFieldVisitor.java

```diff
@@ -41,15 +40,17 @@ public abstract class StoredFieldVisitor {
   protected StoredFieldVisitor() {}
 
   /**
-   * Expert: Process a binary field directly from the {@link DataInput}. Implementors of this method
-   * must read {@code length} bytes from the given {@link DataInput}. The default implementation
-   * reads all byes in a newly created byte array and calls {@link #binaryField(FieldInfo, byte[])}.
+   * Expert: Process a binary field directly from the {@link StoredFieldDataInput}. Implementors of
+   * this method must read {@code StoredFieldDataInput#length} bytes from the given {@link
+   * StoredFieldDataInput}. The default implementation reads all byes in a newly created byte array
```

Comment:
it is an existing typo.

```suggestion
   * StoredFieldDataInput}. The default implementation reads all bytes in a newly created byte array
```